src/app/sign-in/sign-in.page.ts

```diff
@@ -0,0 +1,145 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { AuthService } from '../services/auth';
+import { NgForm } from '@angular/forms';
+import { LoadingController, NavController } from '@ionic/angular';
+import { ApiService } from '../services/api';
+import { ConnectionService } from '../services/connection';
+import { CustomersService } from '../services/customers';
+import { MessagesService } from '../services/messages';
+import { UtilsService } from '../services/utils';
+import { Router } from '@angular/router';
+import { User } from '../data/user.interface';
+import { Customer } from '../data/customer.interface';
+import { OfflineDataPersisterService } from '../services/offline_data_persistence/offline_data_persister';
+
+@Component({
+  selector: 'app-sign-in',
+  templateUrl: './sign-in.page.html',
+  styleUrls: ['./sign-in.page.scss'],
+})
+export class SignInPage implements OnInit {
+  cities: Customer[] = [
+    { name: 'Treinamento', url: 'https://treinamento.idiario.net.br', support_url: '' }
```

Comment:
Isso pode ser um buscado de um arquivo `.ts` padrão e caso não exista podemos usar um fallback para outro arquivo com:

```suggestion
    { name: 'i-Diário', url: 'http://localhost:3000', support_url: '' }
```